.ci/wercker.sh

```diff
@@ -61,10 +61,10 @@ no-error-pgjdbc)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/pgjdbc/pgjdbc.git
+  checkout_from https://github.com/Abhishek-kumar09/pgjdbc.git
```

Comment:
is this line should be like this? I think there should be no user forks

Reply:
We can keep it for a little, as @Abhishek-kumar09 become frequent contributor.
PR to pgjdbc will be created after merge.

There are two updates to this project for indentation. So we will provide single PR to them.